entries/s/sncf-connect.com.json

```diff
@@ -0,0 +1,15 @@
+{
+  "SNCF Connect": {
+    "domain": "sncf-connect.com",
+	  "additional-domains": [
+	    "oui.sncf"
+	  ],
+    "tfa": [
+      "email"
+    ],
+	  "documentation": "https://www.sncf-connect.com/en-en/help/your-customer-account",
+    "keywords": [
+      "transport"
+    ]
```

Comment:
```suggestion
    ],
    "regions": [
      "fr"
    ]
```

Reply:
Only reason I removed that is because the website is for France and other countries in Europe (see region selector at the bottom of their website

Reply:
This seems to be a language selector only, if I understand it correctly.

There are train destinations in other countries, but the target audience seems to be travellers from France.